controller.js

```diff
@@ -48,12 +46,26 @@ exports.getComments = (request, response, next) => {
 
   Promise.all([articleIdCheck, selectComments])
     .then(([article, comments]) => {
-      if (article.length === 0) {
-        return Promise.reject({ status: 404, msg: "Path not found!" });
-      }
       response.status(200).send({ comments });
     })
     .catch((err) => {
       next(err);
     });
 };
+
+exports.postComment = (request, response, next) => {
+  const { article_id } = request.params;
+  const commentToPost = request.body;
+
+  const articleIdCheck = selectArticleById(article_id);
+  const insertedComment = insertComment(article_id, commentToPost);
+
+  Promise.all([articleIdCheck, insertedComment])
```

Comment:
will the insertComment promise resolve without checking the article beforehand, I would think that you would get a psql error, probably a foreign key violation If you tried to adda comment to a review that doesnt exist?